src/assets/sass/protocol/base/elements/_sections.scss

```diff
@@ -23,6 +23,28 @@
     }
 }
 
+.mzp-s-content {
```

Comment:
Sadly the `l` in `mzp-l-content` stands for "layout" not "large". Though, now I kind of wish it did. Here's [an explanation for the namespaces](https://protocol.mozilla.org/docs/naming.html).